gymnasium/utils/env_checker.py

```diff
@@ -165,6 +165,39 @@ def check_reset_options(env: gym.Env):
         )
 
 
+def check_step_return(env: gym.Env, seed=123):
+    """Check that the environment steps dermistically after reset.
+
+    Note: This check assumes that seeded `reset()` is derministic (it must have passed `check_reset_seed`) and that `step()` returns valid values (passed `env_step_passive_checker`).
+    Note: A single step should be enough to assert that the state transition function is deterministic (at least for most environments).
+
+    Author: @Kallinteris-Andreas
```

Comment:
Could you remove, most of the project doesn't include authorship links and if people want to know who wrote stuff can use git blame